napalm_panos/panos.py

```diff
@@ -344,6 +345,28 @@ def rollback(self):
             except:    # noqa
                 ReplaceConfigException("Error while loading backup config")
 
+    def _extract_interface_list(self):
+        try:
+            self.device.op(cmd='<show><interface>all</interface></show>')
+            interfaces_xml = xmltodict.parse(self.device.xml_root())
+            interfaces_json = json.dumps(interfaces_xml['response']['result'])
+            interfaces = json.loads(interfaces_json)
+        except AttributeError:
+            interfaces = {}
+
+        interface_set = set()
+
+        for _, entry in interfaces.items():
```

Comment:
Why not use `values()` ?

Reply:
👍 I was initially extracting both key and val, then changed but forgot to update the generator.